gordon_janitor/main.py

```diff
@@ -180,6 +180,13 @@ def run(config_root):
     loop = asyncio.get_event_loop()
     try:
         loop.run_until_complete(_run(plugins, debug_mode))
+        logging.info('Gordon-janitor run complete.')
+        if 'metrics' in plugin_kwargs:
+            metrics = plugin_kwargs['metrics']
+            loop.run_until_complete(metrics.incr('run-complete'))
+    except Exception as e:
+        logging.error(f'An error occurred during the janitor run: {e}')
+        raise e
```

Comment:
If we're thinking of alerting on not successfully completing a run over some period of time, would it make sense to add a metric here too? We can still alert on the absence of the successful run metric, but it would still go off if the monitoring stack itself is failing for some reason. And so seeing an error metric would give more color. Just a thought.

Reply:
I agree! I've updated the code to add two different error metric scenarios. What do you think?